ern-local-cli/src/commands/cauldron/get/dependency.ts

```diff
@@ -34,9 +40,9 @@ export const commandHandler = async ({
 
   const cauldron = await getActiveCauldron()
   const dependencies = await cauldron.getNativeDependencies(descriptor)
-  for (const dependency of dependencies) {
-    log.info(dependency.toString())
-  }
+  json
+    ? console.log(JSON.stringify(dependencies, null, 2))
```

Comment:
console.log?

Reply:
why not ? ;)

Reply:
I remember we had PR long back to get rid of `console.log`, fail to gather context though.

Reply:
Yes that's correct, `console.log` should not be used indeed, but rather use `log.` functions. 
That being said, as noted, this is a special case due to the fact that `log.` is turned off when `--json` flag is on.
But I can see the confusion, and that's fair, so I'll use `process.stdout.write` instead, so that we can keep the rule of no `console.log` in the code base.

Reply:
done